conf/uvek.conf.part

```diff
@@ -1655,7 +1655,7 @@ source src_ch_bfe_komo_projekte : def_searchable_features
 
 source src_ch_bfe_thermische_netze : def_searchable_features
  {
-     sql_db = uvek_dev
+     sql_db = uvek_${DBSTAGING}
```

Comment:
There's one `_dev` left that is not replaced by `${DBSTAGING}` in line 1676, is this by intention or was it forgotton?

Reply:
😨 forgotten
thank you!